swagger/security_aad.json

```diff
@@ -0,0 +1,35 @@
+﻿{
```

Comment:
nit: name the files `kebab-case` to be consistent

Reply:
got it